### PR TITLE
add icon for helpwave tasks

### LINKED ISSUE #155: Icon and Splash Screen
For the helpwave tasks app add, whenDesign is ready:
- [ ] Splash Screen
- [ ] App Icon

### PR DESCRIPTION
**Pull request by DasProffi**
closes #155

### CODE REVIEW

**Review by florian-sabonchi — APPROVED** [review]

File: apps/tasks/pubspec.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 name: tasks
-description: A new Flutter project.
-# The following line prevents the package from being accidentally published to
-# pub.dev using `flutter pub publish`. This is preferred for private packages.
-publish_to: 'none' # Remove this line if you wish to publish to pub.dev
+description: The helpwave app for managing taks in a hospital.
```
Comment (use-to) [inline apps/tasks/pubspec.yaml:2]: Typo

File: apps/tasks/web/index.html [hunk 2]
```diff
@@ -55,5 +60,6 @@
       });
     });
   </script>
-</body>
-</html>
+
+
+</body></html>
```
Comment (use-to) [inline apps/tasks/web/index.html:65]: Weird formatting, do we really need flutter web?

**Review by use-to — CHANGES_REQUESTED** [review]

File: apps/tasks/web/index.html (hunk 2 above)
Comment (DasProffi) [reply apps/tasks/web/index.html:65]: it was generated at the initialization, therefore not part of this issue, but good point

Comment (use-to) [reply apps/tasks/web/index.html:65]: Moved to internal chat!

**Review by use-to — APPROVED** [review]

**Pull request merged**